README.md

```diff
@@ -37,6 +37,7 @@ const config = {
   savePath: './static/data',
   compressJSON: true, // setting this to false may help debugging :-)
   perPage: 5000, // arbitrary
+  postsPerPage: 10, // set for paginated posts (default will automatically be 10)
```

Comment:
post = wp post object (post, page, etc.)
or
post = wp post type `post` (if yes, maybe use `itemsPerPage`

Reply:
anyway check if the example is correct in readme :)

Reply:
👍